src/Integrations/Airtable/AirtableClient.php

```diff
@@ -348,24 +348,26 @@ private function getAirtableLists()
 	 * Prepare params
 	 *
 	 * @param array<string, mixed> $params Params.
+	 * @param string $formId FormId value.
 	 *
 	 * @return array<string, mixed>
 	 */
-	private function prepareParams(array $params): array
+	private function prepareParams(array $params, string $formId): array
 	{
 		$output = [];
 
 		// Map enrichment data.
 		$params = $this->enrichment->mapEnrichmentFields($params);
 
-		// Remove unecesery params.
-		$params = Helper::removeUneceseryParamFields($params);
-
+		// Filter params.
 		$filterName = Filters::getFilterName(['integrations', SettingsAirtable::SETTINGS_TYPE_KEY, 'prePostParams']);
 		if (\has_filter($filterName)) {
-			$params = \apply_filters($filterName, $params) ?? [];
+			$params = \apply_filters($filterName, $params, $formId) ?? [];
 		}
 
+		// Remove unecesery params.
+		$params = Helper::removeUneceseryParamFields($params);
```

Comment:
```suggestion
		$params = Helper::removeUnnecessaryParamFields($params);
```
For some other day 😄